chain/cosmos/cosmos_chain.go

```diff
@@ -1044,323 +1034,6 @@ func (c *CosmosChain) Start(testName string, ctx context.Context, additionalGene
 	return testutil.WaitForBlocks(ctx, 2, c.getFullNode())
 }
 
-// Bootstraps the provider chain and starts it from genesis
-func (c *CosmosChain) StartProvider(testName string, ctx context.Context, additionalGenesisWallets ...ibc.WalletAmount) error {
```

Comment:
This is all just copy pasted to the new ics.go file. Easier future maintenance since it typically lags major versions